docs/pushing-images.md

```diff
@@ -0,0 +1,23 @@
+# Pulling OperatorBundles from quay.io
```

Comment:
I see that a document was added for pulling images, should we include a document for pushing images? Might not be necessary with the work being done on [operator-courier](https://github.com/operator-framework/operator-courier)

Reply:
All I did was move this content from the readme to here. If it's obsoleted later on (or now), it can be removed.